apps/meteor/server/models/raw/VideoConference.ts

```diff
@@ -136,6 +137,13 @@ export class VideoConferenceRaw extends BaseRaw<VideoConference> implements IVid
 		return (await this.insertOne(call)).insertedId;
 	}
 
+	public async createVoIP(call: InsertionModel<IVoIPVideoConference>): Promise<string | undefined> {
+		const { externalId, ...data } = call;
+
+		const doc = await this.findOneAndUpdate({ externalId }, { $set: data }, { upsert: true, returnDocument: 'after' });
+		return doc.value?._id;
```

Comment:
if we are returning the `_id` only, why not use `insertOne` and return the `insertedId`

Reply:
yeah but in this case `externalId` is not unique, so i assume this was to provide some sort of uniqueness without making it actually unique (cause then other videoconf documents with no externalId would complain).

Was planning to use a partialFilterExp to make the unique only when externalId was present, but as i'm not very familiar with videoconf decided to leave that improvement for later. wdyt?